lib/developer_portal/lib/site_account_support.rb

```diff
@@ -52,7 +52,7 @@ def buyer_domain?
   end
 
   def admin_domain?
-    current_account.internal_self_domain == internal_host
+    current_account.external_admin_domain == internal_host
```

Comment:
This should be `internal_admin_domain`, as `internal_host` is the host after translation to internal.